components/rootless_studio/default/profile

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# Setting the user file-creation mask (umask) to 022 ensures that newly created
+# files and directories are only writable by their owner, but are readable and
+# executable by anyone (assuming default modes are used by the open(2) system
+# call, new files will end up with permission mode 644 and directories with
+# mode 755).
+umask 022
+
+export PATH=$PATH:/bin
+STUDIO_TYPE="default"
+# Colorize ls by default
+if command -v dircolors > /dev/null; then
+  eval "$(dircolors -b)"
+fi
+alias ls="ls --color=auto"
+alias ll="ls -l"
+alias la="ls -al"
+
+# Set a prompt which tells us what kind of Studio we're in
+if [ "${HAB_NOCOLORING:-}" = "true" ]; then
+  PS1='[\#]'${HAB_STUDIO_BINARY+[HAB_STUDIO_BINARY]}[${STUDIO_TYPE:-unknown}':\w:`echo -n $?`]\$ '
+else
+  case "${TERM:-}" in
+  *term | xterm-* | rxvt | screen | screen-*)
+    PS1='\[\e[0;32m\][\[\e[0;36m\]\#\[\e[0;32m\]]${HAB_STUDIO_BINARY+[\[\e[1;31m\]HAB_STUDIO_BINARY\[\e[0m\]]}['${STUDIO_TYPE:-unknown}':\[\e[0;35m\]\w\[\e[0;32m\]:\[\e[1;37m\]`echo -n $?`\[\e[0;32m\]]\$\[\e[0m\] '
+    ;;
+  *)
+    PS1='[\#]'${HAB_STUDIO_BINARY+[HAB_STUDIO_BINARY]}[${STUDIO_TYPE:-unknown}':\w:`echo -n $?`]\$ '
+    ;;
+  esac
+fi
+
+# Colorize grep/egrep/fgrep by default
+alias grep='grep --color=auto'
+alias egrep='egrep --color=auto'
+alias fgrep='fgrep --color=auto'
+
+# Set TERMINFO so hab can give us a delightful experience.
+export TERMINFO=$(hab pkg path core/ncurses)/share/terminfo
+
+emacs() {
+  if command -v emacs > /dev/null; then
+    emacs "$@"
+  else
+    mg "$@"
+  fi
+}
```

Comment:
Please add a `vim` function as well.  😆

Reply:
Vim is brought in by default with backline - but you probably already knew that :trollface: